_employment/employers.md

```diff
@@ -34,20 +34,33 @@ There are incentives such as the Work Opportunity Tax Credit (WOTC) that ranges
       <ul class="small-block-grid-1 medium-block-grid-3 cards small">
         <li>
           <a href="/employment/employers/support-veteran-employees/">
-          <h5>Workplace Support</h5>
-          <span>Find ways to make your Veteran employees feel at home.</span></a>
+            <h5>
+              Workplace Support
+            </h5>
+            <span>
+              Find ways to make your Veteran employees feel at home.
+            </span>
+          </a>
         </li>  
 
         <li>
           <a href="/employment/employers/apprenticeship/">
-            <h5>Establish an Apprenticeship and On-The-Job-Resources</h5>
+            <h5>
+              <span>
```

Comment:
You don't need this extra <span> within the h5
![screen shot 2016-04-06 at 8 22 29 pm](https://cloud.githubusercontent.com/assets/1307774/14339671/5208319c-fc35-11e5-9204-68ef3bb94e3e.png)

It can be like the one above for workplace support with just the <h5>title</h5> and the subtitle in the <span>s